react/.storybook/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "plugins": ["@babel/proposal-export-default-from"]
```

Comment:
I think we don't need this file anymore - this was just a step we took in debugging, right?  Babel settings are configured in `main.js`